### DISCUSSION TITLE
Planned date for 1.0.0?

### DISCUSSION THREAD

**Discussion by mr1upmachine** [chartjs/chartjs-plugin-datalabels#226] [Q&A] [answered]
Is there a planned timeline for the 1.0.0 release?

**Comment by simonbrunel** [comment, marked as answer]
@Mr1upMachine Sorry for the late update: I just published [v1.0.0-beta.1](https://github.com/chartjs/chartjs-plugin-datalabels/releases/tag/v1.0.0-beta.1). This version (v1.x) will be the stable release for Chart.js 2.x.

Any help testing this beta is welcome so I can release v1.0.0 soon.

**Reply by simonbrunel** [reply]
[Version 1.0.0](https://github.com/chartjs/chartjs-plugin-datalabels/releases/tag/v1.0.0), compatible with Chart.js 2.x, has been released today!
